### ISSUE TITLE
'systemctl status' shows invalid time under Since

### ISSUE THREAD

**Issue by twojstaryzdomu** [systemd/systemd#20723]
**systemd version the issue has been seen with**
 > 247.9

**Used distribution**
 > Debian

**Linux kernel version used** (`uname -a`)
 > `Linux hostname 5.10.60+ #1449 Wed Aug 25 14:59:20 BST 2021 armv6l GNU/Linux`

**CPU architecture issue was seen on**
 > armhf

**Expected behaviour you didn't see**
 > Didn't see 'systemctl status' (without a service/PID) showing the correct system time under Since.

**Unexpected behaviour you saw**
 > 'systemctl status' (without a service/PID) shows invalid time under Since. The command appears to assume the system has been running since the Epoch. The system clock is correctly set with NTP when the command is ran. 

**Steps to reproduce the problem**
 > `systemctl status`
or
 > `systemctl status --all`

**Additional program output to the terminal or log subsystem illustrating the issue**
```● hostname
    State: running
     Jobs: 0 queued
   Failed: 0 units
    Since: Thu 1970-01-01 00:00:04 GMT; 51 years 8 months ago
...
```

**Comment by poettering** [comment]
my guess is that your system lacks an RTC? we take the timestamp early at boot, and if the system clock isn't set by then it doesn't helper that it might be set later on.

**Comment by twojstaryzdomu** [comment]
Indeed, no RTC or battery backed-up clock installed. The time reported stands in stark contrast with other commands such as `dmesg -T` or `uptime -s`, both of which report the right time. `fake-hwclock` runs early at boot in this environment that is exhibiting the problem but it makes no difference on what `systemctl status` reports.
